trollflow2/tests/test_trollflow2.py

```diff
@@ -2018,7 +2019,8 @@ def test_filepublisher_kwargs_direct_instance_no_nameserver(self):
 
             _ = FilePublisher(port=40000, nameservers=False)
             NoisyPublisher.assert_not_called()
-            Publisher.assert_called_once_with('tcp://*:40000', name='l2processor', min_port=None, max_port=None)
+            assert "tcp://*:40000" in Publisher.mock_calls[0].args
```

Comment:
the uniqueness check of the call is dropped, is this on purpose?

Reply:
Yes. The test result depended on the Posttroll version (with current `main` the `min_port` and `max_port` are not in the default call) so this is more generic.

Reply:
ok, but should we check that it is only called once?

Reply:
Oh, yeah. Will do that.

Reply:
Fixed in https://github.com/pytroll/trollflow2/pull/205/commits/76de724eb4fe201653a40836dee4b279b73b1c5d